concepts/05 UI Components/SpeechToText/00 Overview/10 Key Features.md

```diff
@@ -0,0 +1,14 @@
+- **Text Editor Integration**    
+You can integrate the SpeechToText component with any text input, including other [DevExtreme Components](/Documentation/Guide/{Framework}_Components/DevExtreme_{Framework}_Components/).
+
+- **Language and Localization Support**    
+SpeechToText implements the Web Speech API SpeechRecognition interface, which supports transcription in multiple languages, including English, Spanish, Hindi, and Portuguese. Supported languages may vary depending on the browser.
+
+- **Configurable Transcription**    
+You can configure [continuous transcription](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/speechRecognitionConfig/#continuous) and [interim transcription results](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/speechRecognitionConfig/#interimResults) to customize SpeechToText transcription behaviour. You can also bias SpeechToText results to a specified [array of words and phrases](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/speechRecognitionConfig/#grammars).
+
+- **Text and Icon Customization**    
+DevExtreme SpeechToText allows you to customize the component [icon](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#startIcon) and [text](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/#startText) in both the initial and "listening" state.
+
+- **Custom Speech Recognizer Support**    
+You can integrate a [custom speech recognizer](/Documentation/ApiReference/UI_Components/dxSpeechToText/Configuration/customSpeechRecognizer/) and use SpeechToText as a UI-only component.
```

Comment:
We should say a few words about default speech recognition method because it's important info